vespaclient-java/src/main/sh/vespa-visit.sh

```diff
@@ -74,16 +74,30 @@ findhost
 
 # END environment bootstrap section
 
+# Allow -Xmx to be specified in args
+for arg in "$@"; do
+  shift
+  if case $arg in -Xmx*) true;; *) false;; esac; then # portable if [string begins with]
+    export Xmx=$arg
+  else
+    set -- "$@" "$arg"
+  fi
+done
+
+if [ "${VESPA_LOG_LEVEL}" = "" ]; then
+    export VESPA_LOG_LEVEL=error,warning
+fi
+
 if [ "${VESPA_LOG_LEVEL}" = "" ]; then
     export VESPA_LOG_LEVEL=error,warning
 fi
 
-export MALLOC_ARENA_MAX=1 #Does not need fast allocation
-exec java \
+export MALLOC_ARENA_MAX=1 # Does not need fast allocation
+echo exec java \
```

Comment:
looks like you forgot to remove echo here

Reply:
Thanks, fixed.